### DISCUSSION TITLE
Ratio can be Uploaded Size / Torrent Size ?

### DISCUSSION THREAD

**Discussion by Alexis-Fayeulle** [transmission/transmission#2873] [Ideas]
Hello

I use transmission on web interface
How to configure the ratio or have another field to have the calculation "Uploaded size / Torrent size" ? (in the ratio or in another variable)

and if it does not exist
has it already been requested?
and if so, what is the decision?

Thanks for your replies

**Comment by josephj11** [comment]
This is a pet peeve of mine. It appears  that Transmission calculates ratios based on downloaded size which is often significantly larger than torrent size. This doesn't make sense to me. is there a reason it is configured this way?

I don't use the web interface. This occurs on the local torrent status windows and has been calculated this way for years.

**Comment by Alexis-Fayeulle** [comment]
Oh not that torrent size  
By "Torrent size" i mean, every activated element in torrent in interface

Example:  
=> example.torrent 100o
```
foo.txt  1Ko
bar.mp3  2Mo
baz.mp4  25Mo
```
Torrent size = `1Ko + 2Mo + 25Mo` (and not 100o)
If everything is check to be downloaded  
if i have disabled `baz.mp4` its `1Ko + 2Mo`

**Comment by tearfur** [comment]
I'd suggest discussing this in #6087.
